index.html

```diff
@@ -3514,6 +3514,58 @@ <h3>Reserved Extension Points</h3>
         </p>
 
       </section>
+
+      <section class="normative">
+        <h3>Ecosystem Compatibility</h3>
+
+        <p>
+There are a number of digital credential formats that do not natively use the
+data model provided in this document, but are aligned with a number of concepts
+in this specification. At the time of publication, examples of these digital
+credential formats include pure JSON Web Tokens (JWTs), CBOR Web Tokens (CWTs),
+ISO-18013-5 (mDLs), and Authentic Chained Data Containers (ACDCs).
+        </p>
+
+        <p>
+If conceptually aligned digital credential formats can be transformed to the
+data model according to the rules provided in this section, they are considered
+<em>"compatible with the Verifiable Credentials ecosystem"</em>, and the result
+of the transformation, which is a <a>conforming document</a>, is a
+<a>verifiable credential</a>. Specifications that contain transformations to the
+data model described by this specification:
+        </p>
+
+        <ul>
+          <li>
+MUST identify if the transformation to this data model is uni-directional or
+bi-directional.
+          </li>
+          <li>
+MUST preserve the `@context` values when performing bi-directional
+transformation.
+          </li>
+          <li>
+MUST result in a <a>conforming document</a>.
+          </li>
+          <li>
+MUST specify a media type for the input document, which will be transformed into
+a <a>conforming document</a>.
+          </li>
+          <li>
+SHOULD provide a test suite that demonstrates that the transformation algorithm
```

Comment:
I also like this one, we could consider publish a set of test vectors to map into... that would make this even easier for implementers to achieve.

Reply:
Test suite before specification document that describes the transformation?

Reply:
I mean that the working group can publish a collection of `vc+ld+json` and `vp+ld+json` and then any other media types can show how they map to them... if they want to ... in documents... outside this working group.

Which raises the obvious question of, if the mapping needs to be `injective` or not...  https://en.wikipedia.org/wiki/Injective_function

Reply:
FWIW, I don't think it MUST be injective... but it probably SHOULD be injective (otherwise, you might lose important information leading to a less ideal experience w/ the VC version of the digital credential). There are probably also cases where metadata in the originating data format doesn't need to come over to the VC. 

I'm not sure we can say much about this without raising more questions than we answer.